src/commands/dev-env-sync-sql.ts

```diff
@@ -355,22 +328,8 @@ DROP PROCEDURE vip_sync_update_blog_domains;
 		console.log( 'Generating search-replace configuration...' );
 		this.generateSearchReplaceMap();
 
-		try {
-			console.log( 'Running the following search-replace operations on the SQL file:' );
-			for ( const [ domain, landoDomain ] of Object.entries( this.searchReplaceMap ) ) {
-				console.log( `  ${ domain } -> ${ landoDomain }` );
-			}
-
-			await this.runSearchReplace();
-			console.log( `${ chalk.green( '✓' ) } Search-replace operation is complete` );
-		} catch ( err ) {
-			const error = err as Error;
-			await this.track( 'error', {
-				error_type: 'search_replace',
-				error_message: error.message,
-				stack: error.stack,
-			} );
-			exit.withError( `Error replacing domains: ${ error.message }` );
```

Comment:
`DevEnvImportSQLCommand` is already performing `search-replace`, so there's no need to do it twice.